src/test/java/com/checkmarx/ast/ResultTest.java

```diff
@@ -53,9 +53,13 @@ void testResultsSummaryJSON() throws Exception {
     void testResultsStructure() throws Exception {
         List<Scan> scanList = wrapper.scanList("statuses=Completed");
         Assertions.assertTrue(scanList.size() > 0);
-        String scanId = scanList.get(0).getId();
-        Results results = wrapper.results(UUID.fromString(scanId));
-        Assertions.assertEquals(results.getTotalCount(), results.getResults().size());
+        for (Scan scan : scanList) {
+            Results results = wrapper.results(UUID.fromString(scan.getId()));
+            if (results != null && results.getResults() != null) {
+                Assertions.assertEquals(results.getTotalCount(), results.getResults().size());
```

Comment:
Dont we want to break the loop but assert not as part of the loop?  this way we will not pass the assertion.

Reply:
casue as it now, it'll assert only if we are in the if. and we want to make sure that the scanned did have results